wg/survey/2024-operator-survey.md

```diff
@@ -0,0 +1,106 @@
+# Etcd Operator Survey Results
```

Comment:
Nit: etcd is always lowercase.  There are several places in the doc to update this.
```suggestion
# etcd Operator Survey Results
```